src/storage-controller/src/lib.rs

```diff
@@ -1133,18 +1161,22 @@ where
     #[tracing::instrument(level = "debug", skip_all)]
     fn append_table(
         &mut self,
-        commands: Vec<(GlobalId, Vec<Update<Self::Timestamp>>, Self::Timestamp)>,
+        write_ts: Self::Timestamp,
```

Comment:
If a `write_ts` is not required can we make this interface accept standard DD triplets of `(data, time, diff)` and the new upper?